package.json

```diff
@@ -128,6 +128,7 @@
     "angular-ui-sortable": "^0.19.0",
     "angular-ui-validate": "^1.2.3",
     "bootstrap": "^4.3.1",
+    "chromium": "^3.0.1",
```

Comment:
Is this an expected change? Generally, we don't expect any changes this file from non-upgrade PR!

Reply:
@DubeySandeep without chromium it is not possible to print anything while using console.log() on the console while writing custom eslint checks, I've added it so that developers do not face any problem while trying to debug or write a new linter

Reply:
Removed as per offline discussion